### PR TITLE
Updated instructions for updating a cask

### PR DESCRIPTION
**Pull request by vikeri**
- Set variable `github_user`
- Fork homebrew-cask before `cask-repair`

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -31,7 +31,12 @@ cask-repair --help
 
 # use to update <outdated_cask>
 outdated_cask='<the-cask-i-want-to-update>'
+github_user='<your-github-user>'
```
Comment (vitorgalvao) [inline CONTRIBUTING.md]: Do `github_user='<my-github-username>'` instead (see line 18).

**Comment by vitorgalvao** [comment]
`github_user` is repeated from the previous step, but I don’t mind it if you think it looks clearer.

@adidalal If you like what you see, feel free to merge.

**Comment by vikeri** [comment]
I was directed directly to the update task from another project so I never went through the previous tasks and thus never assigned the variable. Couldn't hurt doing it twice I guess?

Updated the PR with `my-github-username` instead though.

**Comment by adidalal** [comment]
Thank you for the contribution. It was merged directly as dda1a8dfe3b1f4c06f5d39a9d837d01f64e0299b to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**